Q4) The director of a state agency believes that the average starting salary for clerical employees in the state
is less than RM30,000 per year. To test her hypothesis, she has collected a simple random sample of 100
starting clerical salaries from across the state and found that the sample mean is RM29,750.

a) State the appropriate null and alternative hypotheses.

Hypothesis claim average starting salary is less than RM30,000 per year

Ho : Average salary >= RM30000
Ha : Average salary < RM30000

b) Assuming the population standard deviation is known to be RM2,500 and the significance level for
the test is to be 0.05, what is the critical value (stated in dollars)?

In [7]:
n = 100 #sample size
sd = 2500 #sample standard deviation
sl = 0.05 #significant level
sm = 29750 #sample mean
h = 30000 #hypothesis mean

In [8]:
import scipy.stats as st 
import math as m

In [9]:
#compute test statistic
value1 = sm - h
value2 = sd / m.sqrt(n)

z = value1 / value2
print(z)

-1.0


c) Referring to your answer in part b, what conclusion should be reached with respect to the null
hypothesis?

In [10]:
st.norm.ppf(sl)

-1.6448536269514729

z(-1.0) > sl(-1.645), therefore Ho should be accepted. There is no sufficient evidence that average starting salry is less than RM30000

d) Referring to your answer in part c, which of the two statistical errors might have been made in
this case? Explain.

type II error have been made as they assume Ho is false when in fact it is true.